test/legacy_test/test_svdvals_op.py

```diff
@@ -0,0 +1,158 @@
+        self.check_grad(['x'], ['s'], numeric_grad_delta=0.001, check_pir=True)
+
+
+class TestSvdvalsBatched(TestSvdvalsOp):
+    """Test svdvals operation with batched input."""
+
+    def init_data(self):
+        """Generate batched input matrix."""
+        self._input_shape = (10, 3, 6)
+        self._input_data = np.random.random(self._input_shape).astype("float64")
+
+        self._output_data = np.linalg.svd(
+            self._input_data, compute_uv=False, hermitian=False
+        )
+
+        self.inputs = {'x': self._input_data}
+        self.outputs = {"s": self._output_data}
+
+
+@skip_check_grad_ci(
+    reason="'check_grad' on singular values is not required for svdvals."
+)
+class TestSvdvalsBigMatrix(TestSvdvalsOp):
+    def init_data(self):
+        """Generate large input matrix."""
+        self._input_shape = (200, 300)
+        self._input_data = np.random.random(self._input_shape).astype("float64")
+        self._output_data = np.linalg.svd(
+            self._input_data, compute_uv=False, hermitian=False
+        )
+        self.inputs = {'x': self._input_data}
+        self.outputs = {'s': self._output_data}
+
+    def test_check_grad(self):
+        pass
+
+
+class TestSvdvalsAPI(unittest.TestCase):
+    def setUp(self):
+        np.random.seed(1024)
+        self.x_np = np.random.uniform(-3, 3, [10, 12]).astype('float32')
+        self.place = (
+            paddle.CUDAPlace(0)
+            if paddle.is_compiled_with_cuda()
+            else paddle.CPUPlace()
+        )
+
+    def test_dygraph_api(self):
+        with dygraph_guard():
+            x = paddle.to_tensor(self.x_np)
+            # Test dynamic graph for svdvals
+            s = paddle.linalg.svdvals(x)
+            np_s = np.linalg.svd(self.x_np, compute_uv=False, hermitian=False)
+            self.assertTrue(np.allclose(np_s, s.numpy(), rtol=1e-6))
+
+            # Test with reshaped input
+            x_reshaped = x.reshape([-1, 12, 10])
+            s_reshaped = paddle.linalg.svdvals(x_reshaped)
+            np_s_reshaped = np.array(
+                [
+                    np.linalg.svd(matrix, compute_uv=False, hermitian=False)
+                    for matrix in self.x_np.reshape([-1, 12, 10])
+                ]
+            )
+            self.assertTrue(
+                np.allclose(np_s_reshaped, s_reshaped.numpy(), rtol=1e-6)
+            )
+
+    def test_static_api(self):
+        with static_guard():
+            with paddle.static.program_guard(
+                paddle.static.Program(), paddle.static.Program()
+            ):
+                x = paddle.static.data('x', [10, 12], dtype='float32')
+                s = paddle.linalg.svdvals(x)
+                exe = paddle.static.Executor(self.place)
+                res = exe.run(feed={'x': self.x_np}, fetch_list=[s])
+
+        np_s = np.linalg.svd(self.x_np, compute_uv=False, hermitian=False)
+        for r in res:
+            self.assertTrue(np.allclose(np_s, r, rtol=1e-6))
```

Comment:
下个PR把所有的assertTrue(allclose)用np.testing.assert_allclose代替，这样错误信息会更清晰
```suggestion
           np.testing.assert_allclose(np_s, r, rtol=1e-6))
```

Reply:
> 下个PR把所有的assertTrue(allclose)用np.testing.assert_allclose代替，这样错误信息会更清晰

了解